schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -197,6 +197,12 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2015-02-01-preview/Microsoft.Logic.json#/resourceDefinitions/workflows" },
                   { "$ref": "https://schema.management.azure.com/schemas/2016-06-01/Microsoft.Logic.json#/resourceDefinitions/workflows" },
                   { "$ref": "https://schema.management.azure.com/schemas/2016-06-01/Microsoft.Logic.json#/resourceDefinitions/integrationAccounts" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2016-06-01/Microsoft.Logic.json#/resourceDefinitions/integrationAccounts/agreements" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2016-06-01/Microsoft.Logic.json#/resourceDefinitions/integrationAccounts/certificates" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2016-06-01/Microsoft.Logic.json#/resourceDefinitions/integrationAccounts/maps" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2016-06-01/Microsoft.Logic.json#/resourceDefinitions/integrationAccounts/partners" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2016-06-01/Microsoft.Logic.json#/resourceDefinitions/integrationAccounts/schemas" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2016-06-01/Microsoft.Logic.json#/resourceDefinitions/integrationAccounts/assemblies" },
```

Comment:
>integrationAccounts/assemblies [](start = 126, length = 30)

do you mean integrationAccounts_assemblies?

Reply:
Thanks, Fixed the issue and updated PR.